packages/remediations/src/steps/IssueTable.js

```diff
@@ -34,7 +34,7 @@ function buildRows(issues, state, getResolution) {
 
 function issueType (id) {
     switch (id.split(':')[0]) {
-        case 'advisor': return 'Insights';
+        case 'advisor': return 'Advisor';
```

Comment:
Shouldn't this be Recommendations? https://github.com/RedHatInsights/insights-remediations-frontend/commit/bee008e234725de2c153c814084489958ec9721b#diff-f473a5177ca02936b12fdd09b3d887f3L80

Reply:
@Robcwilliams said `Advisor` in https://projects.engineering.redhat.com/browse/RHCLOUD-5878